app/Http/Controllers/ConferencesController.php

```diff
@@ -47,28 +47,34 @@ public function index(Request $request)
 
         switch ($request->input('sort')) {
             case 'date':
-                $conferences = $conferences->sortBy(function (Conference $model) {
-                    return $model->starts_at;
-                });
+                $conferences = $conferences->sortBy->starts_at;
                 break;
-            case 'closing_next':
-                // Forces closed CFPs to the end. I feel dirty. Even dirtier with the 500 thing.
-                $conferences = $conferences
-                    ->sortBy(function (Conference $model) {
-                        if ($model->cfp_ends_at > Carbon::now()) {
-                            return $model->cfp_ends_at;
-                        } elseif ($model->cfp_ends_at === null) {
-                            return Carbon::now()->addYear(500);
-                        } else {
-                            return $model->cfp_ends_at->addYear(1000);
-                        }
-                    });
+            case 'opening_next':
+                // Force CFPs with no CFP start date to the end
+                $conferences = $conferences->sortBy(function ($conference) {
+                    return isset($conference->cfp_starts_at) ? $conference->cfp_starts_at : Carbon::now()->addCentury();
+                });
                 break;
             case 'alpha':
-                // Pass through
+                $conferences = $conferences->sortBy->title;
+                break;
+            case 'closing_next':
+                // pass through
             default:
-                $conferences = $conferences->sortBy(function (Conference $model) {
-                    return strtolower($model->title);
+                // Forces closed CFPs to the end.
+                $conferences = $conferences->sortBy(function ($conference) {
```

Comment:
I think we said we were going to simplify this one down too, right?

Reply:
Yep, thanks for catching that!  The default sort now sorts by `cfp_ends_at`, whether it is in the past or future.  Conferences with a `null` `cfp_ends_at` will be sorted to the end.